src/views/nodes-list.njk

```diff
@@ -2,13 +2,20 @@
 
 {% block content %}
 <main>
+  <header class="main-header">
+    <div class="main-title">
+      <h1>All the nodes</h1>
```

Comment:
Again, doesn't quite sit in my English brain :) Maybe `Nodes`, `Nodes list` or `List of all nodes`?